test/conformance/tests/sriov_operator.go

```diff
@@ -825,6 +825,22 @@ var _ = Describe("[sriov] operator", func() {
 	})
 
 	Describe("Custom SriovNetworkNodePolicy", func() {
+		var vfioNode string
+		var vfioNic sriovv1.InterfaceExt
+		execute.BeforeAll(func() {
+			findVfioSriovDevice := func(vendorID string) (string, sriovv1.InterfaceExt) {
```

Comment:
lets move this to https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/test/util/cluster/cluster.go#L99